dbt/models/eia861/core_eia861__yearly_advanced_metering_infrastructure/schema.yml

```diff
@@ -11,15 +11,44 @@ sources:
                 partition_expr: "EXTRACT(YEAR FROM report_date)"
         columns:
           - name: advanced_metering_infrastructure
+            data_tests:
+              - dbt_expectations.expect_column_min_to_be_between:
+                  arguments:
+                    min_value: 0.0
           - name: automated_meter_reading
+            data_tests:
+              - dbt_expectations.expect_column_min_to_be_between:
+                  arguments:
+                    min_value: 0.0
           - name: balancing_authority_code_eia
           - name: customer_class
           - name: daily_digital_access_customers
+            data_tests:
+              - dbt_expectations.expect_column_min_to_be_between:
+                  arguments:
+                    min_value: 0.0
           - name: direct_load_control_customers
+            data_tests:
+              - dbt_expectations.expect_column_min_to_be_between:
+                  arguments:
+                    min_value: 0.0
           - name: energy_served_ami_mwh
+            data_tests:
+              - dbt_expectations.expect_column_min_to_be_between:
+                  arguments:
+                    min_value: 0.0
           - name: entity_type
           - name: home_area_network
+            data_tests:
+              - dbt_expectations.expect_column_min_to_be_between:
+                  arguments:
+                    min_value: 0.0
           - name: non_amr_ami
+            data_tests:
+              - dbt_expectations.expect_column_min_to_be_between:
+                  arguments:
+                    min_value: 0.0
+                    error_if: "<1"
```

Comment:
This seems like it might be backwards. It says "Treat this as an error if there's LESS than 1 record that violates the condition."